deploy-manage/tools/snapshot-and-restore.md

```diff
@@ -36,4 +36,167 @@ $$$snapshot-restore-version-compatibility$$$
 
 $$$snapshots-shard-allocation$$$
 
-$$$fn-archive$$$
\ No newline at end of file
+$$$fn-archive$$$
```

Comment:
This PR needs all of the commented out notes above removed. it also needs all of the anchors to be associated with the relevant content

Reply:
this page is also missing `applies` front matter.